app.go

```diff
@@ -127,8 +128,12 @@ func NewHeadscale(cfg *Config) (*Headscale, error) {
 			cfg.DBuser,
 		)
 
-		if !cfg.DBssl {
-			dbString += " sslmode=disable"
+		if s, err := strconv.ParseBool(cfg.DBssl); err == nil {
+			if !s {
+				dbString += " sslmode=disable"
+			}
```

Comment:
Do we need the inverse of this?

If `s` is true, do we need to add `sslmode=enable` ?

```suggestion
		if sslEnabled, err := strconv.ParseBool(cfg.DBssl); err == nil {
			if !sslEnabled {
				dbString += " sslmode=disable"
			}
```

Reply:
`sslmode=on` is the default behavior so we'll be fine without it.